autogluon/task/tabular_prediction/predictor.py

```diff
@@ -622,6 +622,49 @@ def feature_importance(self, dataset=None, model=None, features=None, feature_st
 
         return self._learner.get_feature_importance(model=model, X=dataset, features=features, feature_stage=feature_stage, subsample_size=subsample_size, silent=silent)
 
+    def persist_models(self, models=None, with_ancestors=True, max_memory=0.1) -> list:
+        """
+        Persist models in memory for reduced inference latency. This is particularly important if the models are being used for online-inference where low latency is critical.
+        If models are not persisted in memory, they are loaded from disk every time they are asked to make predictions.
+
+        Parameters
+        ----------
+        models : list of str or str, default = None
+            Model names of models to persist.
+            If None then all models are persisted.
+            If 'best' then the model with the highest validation score is persisted.
```

Comment:
Shouldn't the default be 'best'?  I imagine this `persist_models()` is primarily used before inference

Reply:
May also clarify: "If 'best' then the model with the highest validation score is persisted (this is the model used for prediction by default)."

Reply:
added